gensim/test/test_fasttext_wrapper.py

```diff
@@ -363,8 +355,8 @@ def testPersistenceForOldVersions(self):
             1.19031894, 2.01627707, 1.98942184, -1.39095843, -0.65036952])
         self.assertTrue(numpy.allclose(loaded_model["the"], in_expected_vec, atol=1e-4))
         # out-of-vocab word
-        out_expected_vec = numpy.array([-1.34948218, -0.8686831, -1.51483142, -1.0164026, 0.56272298,
-            0.66228276, 1.06477463, 1.1355902, -0.80972326, -0.39845538])
+        out_expected_vec = numpy.array([-0.33959097, -0.21121596, -0.37212455, -0.25057459, 0.11222091,
```

Comment:
This is probably not correct, the vector seems to differ between different Python versions (the above was with 3.6)